CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ service runtime.
 For instance, fixing the documentation and lintin SHOULD not be
 included in the changelog document.
 
+## [Unreleased] 
+
+- Added in travisci and GolangCI linter
+- Fix vet warnings
+
```

Comment:
- Added json.Marshaler and json.Unmarshaler interfaces
- Added current market price for definite quantity calculation

Reply:
Done